wiki/Skinning/en.md

```diff
@@ -1,26 +1,30 @@
 # Skinning
 
-*See also: [Skinning Tutorial](/wiki/Skinning/Tutorial).*
+Skinning is one of the fundamentals of all osu! game modes. It enables players to derive from the original skinning elements to create their own!
+
+Skins vary from for-fun to a central theme (e.g. anime or group of people) to those that are nigh impossible to play with. Another type of skin are PRO skins, where they maximize playfield visibility and minimize visual obtrusions (e.g. replacing hitbursts with blank images).
```

Comment:
from for-fun, through a central theme (e.g. anime or group of people) to those that are nigh impossible to play with